pkg/visor/visor.go

```diff
@@ -401,18 +401,16 @@ func (node *Node) Apps() []*AppState {
 
 // StartApp starts registered App.
 func (node *Node) StartApp(appName string) error {
-	for _, app := range node.appsConf {
-		if app.App == appName {
-			startCh := make(chan struct{})
-			go func(app AppConfig) {
-				if err := node.SpawnApp(&app, startCh); err != nil {
-					node.logger.Warnf("Failed to start app %s: %s", appName, err)
-				}
-			}(app)
-
-			<-startCh
-			return nil
-		}
+	if app, isAppExist := node.appsConf[appName]; isAppExist {
```

Comment:
I would prefer to return early if an app doesn't exist to reduce indentation. We can also rename `isAppExist` to `ok` because it's a common way in Go to check if an element exists in a collection.
```golang
app, ok := node.appsConf[appName]
if !ok {
	return ErrUnknownApp
}

// start app
```

Reply:
yeah, agree.. I was gonna use ok back then, but hestitate, because of the readiblity for programmer that not familiar with go.. i'll change it in a minute..

Reply:
No problem, thanks